apps/web/src/pages/EmployeeProfile/components/NextRoleSection/Display.tsx

```diff
@@ -0,0 +1,163 @@
+      data-h2-gap="base(x1)"
+    >
+      {hasAnyEmptyFields({
+        nextRoleClassification,
+        nextRoleTargetRole,
+        nextRoleJobTitle,
+        nextRoleCommunity,
+        nextRoleWorkStreams,
+        nextRoleDepartments,
+        nextRoleAdditionalInformation,
+      }) && (
+        <>
+          <Well>
+            {intl.formatMessage({
+              defaultMessage:
+                'There are currently unanswered questions in this section. Use the "Edit" button to review and answer any required fields.',
+              id: "9Y3w6l",
+              description: "Message for unanswered questions in this section",
+            })}
+          </Well>
+          <CardSeparator data-h2-margin="base(0)" />
+        </>
+      )}
+      <div
+        data-h2-display="base(grid)"
+        data-h2-gap="base(x1)"
+        data-h2-grid-template-columns="base(repeat(1, 1fr)) p-tablet(repeat(2, 1fr)) "
+      >
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(
+            employeeProfileMessages.targetClassificationGroup,
+          )}
+        >
+          {nextRoleClassification?.group
+            ? nextRoleClassification.group
+            : notProvided}
+        </ToggleForm.FieldDisplay>
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(
+            employeeProfileMessages.targetClassificationLevel,
+          )}
+        >
+          {nextRoleClassification?.level
+            ? nextRoleClassification.level.toString().padStart(2, "0")
+            : notProvided}
+        </ToggleForm.FieldDisplay>
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(employeeProfileMessages.targetRole)}
+        >
+          {nextRoleTargetRoleOther ??
+            nextRoleTargetRole?.label.localized ??
+            notProvided}
+        </ToggleForm.FieldDisplay>
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(employeeProfileMessages.jobTitle)}
+        >
+          {nextRoleJobTitle ? nextRoleJobTitle : notProvided}
+        </ToggleForm.FieldDisplay>
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(employeeProfileMessages.community)}
+          data-h2-grid-column="l-tablet(span 2)"
+        >
+          {nextRoleCommunity?.name?.localized
+            ? nextRoleCommunity.name.localized
+            : notProvided}
+        </ToggleForm.FieldDisplay>
+        <ToggleForm.FieldDisplay
+          label={intl.formatMessage(employeeProfileMessages.workStreams)}
+          data-h2-grid-column="l-tablet(span 2)"
+        >
+          {nextRoleWorkStreams?.length ? (
+            <ul
+              data-h2-margin-bottom="base:selectors[>li:not(:last-child)](x.125)"
+              data-h2-padding-left="base(x1)"
+            >
+              {nextRoleWorkStreams.map((workStream) => (
+                <li key={workStream.id}>{workStream?.name?.localized}</li>
+              ))}
+            </ul>
+          ) : (
+            notProvided
```

Comment:
Will a community always have one or more workstreams? If a community could have zero stream options, that effects copy and validator logic

Reply:
Hmm, I'm not sure but we certainly don't enforce that in our API.  I've added handling to the validation and displays for it.
[handle no possible work streams in community](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12844/commits/1cdeb854e083541d999d3890e4c1972837d82547)

Reply:
Nice
I noticed it since we seed communities without streams attached and thought it would be really annoying for a user for something to be incomplete yet impossible to complete